text/0123-user-dictionary-settings-api.md

```diff
@@ -0,0 +1,137 @@
+# Dictionary Setting API
+
+## 1. Summary
+
+This specification describes the `dictionary` index setting API endpoints.
+
+## 2. Motivation
+N/A
+
+## 3. Functional Specification
+
+### 3.1. Explanations
+
+The `dictionary` index setting allows the configuration of a list of words for which the segmentation is overriden in search queries and indexing. The dictionary's words contained in a search query will be segmented as defined in the dictionary.
+
+#### 3.1.1. Usage Example
+
+Suppose a database contains books with authors. Some athors contains separators in their names like `J. R. R. Tolkien` or `J. K. Rowling`. To set `J. R. R.` and `J. K.` words in the dictionary, it can be specified the following way.
+
+***Request payload `PUT`- `/indexes/articles/settings/dictionary`***
+```json
+["J. R. R.", "J. K."]
+```
+
+By adding authors names as `J. R. R.` and `J. K.` to the dictionary, Meilisearch consider them as unique words instead of splitting them in several parts.
```

Comment:
```suggestion
By adding authors' names as `J. R. R.` and `J. K.` to the dictionary, Meilisearch considers them as unique words instead of splitting them into several parts.
```